codec/encode.go

```diff
@@ -17,8 +18,12 @@ func Encode(b interface{}) ([]byte, error) {
 		return encodeInteger(int(v))
 	case int64:
 		return encodeInteger(int(v))
+	case string:
+		return encodeByteArray([]byte(v))
```

Comment:
Does this mean string encoding is complete?

Reply:
yes, this should work for strings. needs tests though